src/cache_entry.h

```diff
@@ -35,6 +35,9 @@
 
 namespace triton { namespace core {
 
+// Note: This can be replaced with std::byte starting in c++17
+using Byte = unsigned char;
```

Comment:
`uint8_t`? move to `common.h` maybe

Reply:
Kept as `unsigned char` from offline discussion, moved to `constants.h`